src/script/standard.cpp

```diff
@@ -327,6 +327,128 @@ CScript GetScriptForMultisig(int nRequired, const std::vector<CPubKey>& keys)
     return script;
 }
 
+/*
```

Comment:
Do we need now all these methods? 
I think that it is better to implement a more generic and flexible solution.

Reply:
> Do we need now all these methods?
> I think that it is better to implement a more generic and flexible solution.

Some of them may have a future use but I think these scripts all have a use case while working with shopping applications or likewise. A more generic one will be very much complex to implement RPC functions on them. `escrow` should have it's own RPC call like `createmultisig`.
I actually thought that it won't hurt to have them now and when we put a generic function there, we can remove the replaced ones.

Reply:
yes, but it is better to add them afterward and if it justifies their inclusion
if this development works well, at the end there will not be the need to have those very or too specific methods

Reply:
Ok.

Reply:
> yes, but it is better to add them afterward and if it justifies their inclusion
> if this development works well, in the end there will not be the need to have those very or too specific methods

Ok